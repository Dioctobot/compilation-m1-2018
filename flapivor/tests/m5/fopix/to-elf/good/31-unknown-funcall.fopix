def f(x) = x + 1

def g(x) = 2 * x

def h(x) = call (if x > 0 then &f else &g end) with (x)

val main =
    print_int (h (1));
    print_int (h (0 - 1))