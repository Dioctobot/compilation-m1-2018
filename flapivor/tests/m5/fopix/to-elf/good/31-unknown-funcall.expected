Process exited with status exited(0).
STDOUT:
2-2
STDERR:



